components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.commons/src/main/java/org/wso2/carbon/cluster/coordinator/commons/configs/CoordinationStrategyConfigUtil.java

```diff
@@ -58,11 +58,11 @@ public void setDataSourceConfig(Map<String, String> dataSourceConfig) {
         this.dataSourceConfig = dataSourceConfig;
     }
 
```

Comment:
Can we remove the dataSourceConfig here since it is included in RdbmsConfig now?